pkg/docker/config/config.go

```diff
@@ -201,6 +245,33 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 
 // RemoveAuthentication deletes the credentials stored in auth.json
 func RemoveAuthentication(sys *types.SystemContext, registry string) error {
+	credConfigs, err := sysregistriesv2.GetRegistryCredentialHelpers(sys, registry)
+	if err != nil {
+		return err
+	}
+	if len(credConfigs) > 0 {
+		if err := func(credConfigs []string, registry string) error {
+			var err error
+			for _, credConfig := range credConfigs {
+				if _, err = getAuthFromCredHelper(credConfig, registry); err != nil {
+					if credentials.IsErrCredentialsNotFoundMessage(err.Error()) {
```

Comment:
This is a good reason to try the next helper, not to abort the loop.

Reply:
is it still necessary to check `credentials.IsErrCredentialsNotFoundMessage`? Can it just log "Not logged in with helper" and continue?

Reply:
Ideally we should be fairly quiet about “not found” and “helper not set up”, but at least somewhat louder about any other kinds of errors; it might be even OK to outright fail for other kinds of errors, if we could reliably detect the “not set up” case and not fail on that one.

Reply:
Outstanding / depends on “helper installed but not set up” detection.